sample-loopd.conf

```diff
@@ -0,0 +1,171 @@
+; booleans are always set to false.
+
+; If only one value is specified for an option, then this is also the
+; default value used by loopd. In case of multiple (example) values, the default
+; is explicitly mentioned.
+; If the part after the equal sign is empty then loopd has no default
+; for this option.
+
+[Application Options]
+
+; The network to run on which can be [regtest|testnet|mainnet|simnet].
+; network=mainnet
+
+; Address to listen on for gRPC clients.
+; rpclisten=localhost:11010
+
+; Address to listen on for REST clients.
+; restlisten=localhost:8081
+
+; The value to send in the Access-Control-Allow-Origin header. Header will be
+; omitted if empty.
+; corsorigin=
+
+; The directory for all of loop's data. If set, this option overwrites
+; --datadir, --logdir, --tlscertpath, --tlskeypath and --macaroonpath.
+; loopdir=~/.loop
+
+; Path to configuration file.
+; configfile=~/.loop/mainnet/loopd.conf
+
+; The directory for loopdb.
+; datadir=~/.loop
+
+; The database backend to use for storing all asset related data. Can be
+; [sqlite|postgres]
+; databasebackend=sqlite
+
+; Path to write the TLS certificate for loop's RPC and REST services.
+; tlscertpath=~/.loop/mainnet/tls.cert
+
+; Path to write the TLS private key for loop's RPC and REST services.
+; tlskeypath=~/.loop/mainnet/tls.key
+
+; Adds an extra IP to the generated certificate.
+; tlsextraip=
+
+; Adds an extra domain to the generated certificate.
+; tlsextradomain=
+
+; Re-generate TLS certificate and key if the IPs or domains are changed.
+; tlsautorefresh
+
+; Do not include the interface IPs or the system hostname in TLS certificate,
+; use first --tlsextradomain as Common Name instead, if set.
+; tlsdisableautofill=false
+
+; Loop's TLS certificate validity period in days. 1 year equals 8760h.
+; tlsvalidity=8760h0m0s
+
+; Path to write the macaroon for loop's RPC and REST services if it doesn't
+; exist.
+; macaroonpath=~/.loop/mainnet
+
+; Directory to log output.
+; logdir=~/.loop/logs
+
+; Maximum logfiles to keep (0 for no rotation).
+; maxlogfiles=3
+
+; Maximum logfile size in MB.
+; maxlogfilesize=10
+
+; Logging level for all subsystems {trace, debug, info, warn, error, critical}
+; -- You may also specify <subsystem>=<level>,<subsystem2>=<level>,... to set.
+; debuglevel=
+
+; Maximum cost in satoshis that loopd is going to pay for an LSAT token
+; automatically. Does not include routing fees.
+; maxlsatcost=1000
+
+; Maximum routing fee in satoshis that we are willing to pay while paying for an
+; LSAT token.
+; maxlsatfee=10
+
+; The maximum number of payment parts that may be used for a loop out swap.
+; loopoutmaxparts=5
+
+; The timeout to use for off-chain payments.
+; totalpaymenttimeout=1h0m0s
+
+; The maximum number of times an off-chain payment may be retried.
+; maxpaymentretries=3
+
+; Enable experimental features: taproot HTLCs and MuSig2 loop out sweeps.
+; experimental=false
```

Comment:
I'd leave out the taproot HTLCs and MuSig2 from the doc here, since these are now the default.

Reply:
Can we also remove that flag from the `loop` app? Can do that in a follow-up if needed.